Orange/widgets/data/owrank.py

```diff
@@ -368,31 +371,33 @@ def set_learner(self, scorer, id):
 
     @memoize_method()
     def get_method_scores(self, method):
-        estimator = method.scorer()
-        data = self.data
-        try:
-            scores = np.asarray(estimator(data))
-        except ValueError:
-            log.warning("Scorer %s wasn't able to compute all scores at once",
-                        method.name)
+        # These errors often happen, but they result in nans, which
+        # are handled correctly by the widget
+        with warnings.catch_warnings():
+            warnings.filterwarnings('ignore', '.*true_divide.*')
+            warnings.filterwarnings('ignore', '.*double_scalars.*')
```

Comment:
I think it would be better to solve the problem of warnings at the source, i.e. in the scoring method that produces them, then in potentially several places where they are used.
Also, it is not hard to imagine that a general warning filter for all methods here can cover up an actual problem/bug. If not now, then maybe in the future.